src/VirtoCommerce.PricingModule.Core/Services/IPriceSearchService.cs

```diff
@@ -0,0 +1,6 @@
+namespace VirtoCommerce.PricingModule.Core.Services
+{
+    public interface IPriceSearchService
```

Comment:
: ISearchService<PricesSearchCriteria, PriceSearchResult, Price>

Reply:
PriceSearchService class implements this interface and SearchService<PricesSearchCriteria, PriceSearchResult, Price>...